types/puppeteer/index.d.ts

```diff
@@ -66,6 +139,12 @@ export type PageEvents =
   | "requestfinished"
   | "response";
 
+export type BrowserEvents =
```

Comment:
I can't find any place where these are used. Perhaps this was a leftover of something else?

Reply:
The `BrowserEvents` have the same purpose as the `PageEvents`, they are just references to available events in the `Browser` and `Page` classes. They can be used to make dynamic event handlers.

Technically, neither type is required, I just kept the code standard from the previous version.

Reply:
Makes sense. :+1: